base/logging.cc

```diff
@@ -978,6 +997,7 @@ LogMessage::~LogMessage() {
 #endif
 
       // Crash the process to generate a dump.
+      RecordReplayDynamicPrintLog("ErrorFatal %s:%d %s", file_, line_, str_newline.c_str());
```

Comment:
OK we can merge this but we need (a) a followup issue, and (b) a comment here pointing to that issue.